signbank/dictionary/tests/test_adminviews.py

```diff
@@ -48,11 +51,37 @@ def test_get_csv(self):
         permission = Permission.objects.get(codename='export_csv')
         self.user.user_permissions.add(permission)
         self.user.save()
-        response = self.client.get(reverse('dictionary:admin_gloss_list'), { 'format': 'CSV' })
+        response = self.client.get(reverse('dictionary:admin_gloss_list'), { 'format': 'CSV-standard' })
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.headers['Content-Type'], 'text/csv; charset=utf-8')
         self.assertEqual(response.headers['Content-Disposition'], 'attachment; filename="dictionary-export.csv"')
 
+    def test_get_ready_for_validation_csv(self):
+        """
+        Tests that a CSV file can be successfully downloaded containing glosses that are
+        tagged ready for validation
+        """
+        permission = Permission.objects.get(codename="export_csv")
+        self.user.user_permissions.add(permission)
+        self.user.save()
+
+        response = self.client.get(
+            reverse("dictionary:admin_gloss_list"),
+            {"format": "CSV-ready-for-validation"},
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.headers["Content-Type"], "text/csv; charset=utf-8")
+        self.assertEqual(
+            response.headers["Content-Disposition"],
+            'attachment; filename="ready-for-validation-export.csv"',
+        )
+
+        content = response.content.decode('utf-8')
+        cvs_reader = csv.reader(io.StringIO(content))
+        # this file is empty, so only the headers are present
+        headers = list(cvs_reader)[0]
+        self.assertEqual(['idgloss', 'gloss_main', 'video_url'], headers)
```

Comment:
It would be useful to test the export of a known gloss here - a lot of the CSV processing happens inside the queryset iteration, so by having an empty queryset, we're not testing as much of the process as we may want to.